tailcfg/tailcfg.go

```diff
@@ -1539,6 +1539,26 @@ type PingResponse struct {
 	IsLocalIP bool `json:",omitempty"`
 }
 
+// MapResponse is the value that's sent by the control plane's coordination
+// server whenever there's a update to tell the client. It's the response to a
+// MapRequest.
+//
+// A client can either send a polling or non-polling MapRequest. If the request
+// isn't a long poll, the corresponding MapResponse will be complete. For
```

Comment:
It doesn't talk about the read-only streaming vs write-only updates which don't result in a mapResponse

Reply:
Fleshed out a bit more.